scripts/release/check_commit_metadata.sh

```diff
@@ -55,14 +58,93 @@ main() {
 	security_label=security
 	security_category=security
 
+	# Order is important here, first partial match wins.
+	declare -A humanized_areas=(
+		["agent/agentssh"]="Agent SSH"
+		["coderd/database"]="Database"
+		["enterprise/audit"]="Auditing"
+		["enterprise/cli"]="CLI"
+		["enterprise/coderd"]="Server"
+		["enterprise/dbcrypt"]="Database"
+		["enterprise/derpmesh"]="Networking"
+		["enterprise/provisionerd"]="Provisioner"
+		["enterprise/tailnet"]="Networking"
+		["enterprise/wsproxy"]="Workspace Proxy"
+		[agent]="Agent"
+		[cli]="CLI"
+		[coderd]="Server"
+		[codersdk]="SDK"
+		[docs]="Documentation"
+		[enterprise]="Enterprise"
+		[examples]="Examples"
+		[helm]="Helm"
+		[install.sh]="Installer"
+		[provisionersdk]="SDK"
+		[provisionerd]="Provisioner"
+		[provisioner]="Provisioner"
+		[pty]="CLI"
+		[scaletest]="Scale Testing"
+		[site]="Dashboard"
+		[support]="Support"
+		[tailnet]="Networking"
+	)
+
+	# Get hashes for all cherry-picked commits between the selected ref
+	# and main. These are sorted by commit title so that we can group
+	# two cherry-picks together.
+	declare -A cherry_pick_commits
+	git_cherry_out=$(
+		{
+			git log --no-merges --cherry-mark --pretty=format:"%m %H %s" "${to_ref}...origin/main"
```

Comment:
I'm honestly not sure how we'd do that. Installing a different version of Git than what's available on your system could be PITA. I suppose this would require embracing Nix or something similar?